portfolio/src/main/webapp/index.html

```diff
@@ -50,8 +51,29 @@ <h1 style="background-color:MediumSeaGreen; text-align:center">Joy's Portfolio</
 			<button onclick="addRandomOpinion()">Fact Me 🧠 </button>
 			<div id="opinion-container"></div>
 
-            <button onclick="getDataUsingAsyncAwait()">Data from Servlet</button>
+            <!--<button onclick="getDataUsingAsyncAwait()">Data from Servlet</button>
                   <div id="data-container"></div>
+                  -->
+
+                  
+                 <!-- <button onclick="getdata()">list</button>
+                  <div id="message-container"></div>
+                  -->
+                  
+     
+     <form action="/data" method="POST">
```

Comment:
I would maybe fix the indentation here so the `<input>` element is indented inside the `<form>`, and the closing `</form>` tag lines up with the opening tag?